README.md

```diff
@@ -8,6 +8,28 @@ The software belongs to and is developed by [WIBU-SYSTEMS AG](https://www.wibu.c
 WIBU-SYSTEMS AG allows voraus robotik GmbH to redistribute the runtimes for you,
 our customers, free of charge[^KB-0336].
 
+## Usage
+
+At first the maintainers publickey should be installed as trusted for `apt` packages like this:
+
+```console
+curl -s --compressed https://wibu-packages.vorausrobotik.com/ubuntu/burfeind_jan-niklas.gpg | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/burfeind_jan-niklas.gpg > /dev/null
+```
+
+Afterwords this repo can be added to the `apt` package sources:
```

Comment:
```suggestion
Afterwards this repo can be added to the `apt` package sources:
```

Reply:
resolved in c5402f11ff3e0f513e3ec9489aba27e15d368dbf